client/extensions/woocommerce/store-sidebar/index.js

```diff
@@ -118,3 +123,17 @@ export default class StoreSidebar extends Component {
 		);
 	}
 }
+
+function mapStateToProps( state ) {
+	const currentUser = getCurrentUser( state );
+	const selectedSiteId = getSelectedSiteId( state );
+	const isSingleSite = !! selectedSiteId || currentUser.site_count === 1;
+	const siteId = selectedSiteId || ( isSingleSite && getPrimarySiteId( state ) ) || null;
+	const site = getSite( state, siteId );
```

Comment:
I wonder if we shouldn't think about putting this information at the base of our app, in order to keep the sidebar in sync with the content...